app/main.py

```diff
@@ -1,13 +1,27 @@
-def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+from typing import List, Dict, Any
 
 
-def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+def format_linter_error(error: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
I think that you could avoid so detailed annotation
`list[dict] `would be enough
Then you could delete import of `List`, `Dict` and `Any`